samples/js/picasa/slurp/src/oauth.js

```diff
@@ -93,7 +65,17 @@ function getAuthCodeViaURL(clientId: string, clientSecret: string)
       scope,
       state: secret,
     });
-    console.log(authUrl);
+
+    const rl = readline.createInterface({
+      input: process.stdin,
+      output: process.stdout,
+    });
+    rl.question(
+      `Visit the following URL and paste the code you get:\n\n\t${authUrl}\n\n`,
+      (answer) => {
+        res([answer, redirectUri]);
```

Comment:
now that the redirect URI is a constant, you could define `'urn:ietf:wg:oauth:2.0:oob'` as a global const, then get rid of the need to pass `redirectUri` through the promise resolve.